packages/base/card-api.gts

```diff
@@ -1364,10 +1363,6 @@ class LinksToMany<FieldT extends CardDefConstructor>
             `linksToMany field '${this.name}' cannot deserialize a list of resource ids`,
           );
         }
-        if (isLocalResourceID(value.data)) {
-          throw new Error(`lid is not supported in deserialization`);
-        }
```

Comment:
this logic is incorrect. its not that lid's are not supported in deserialization, it's that they are _ignored_. we should not being making type assertions like this as `lid` is a supported member of the JSON API spec.